classes/class-wc-connect-settings-pages.php

```diff
@@ -66,7 +66,7 @@ public function output_shipping_settings_screen() {
 			global $hide_save_button;
 			$hide_save_button = true;
 
-			if ( WC_Connect_Jetpack::is_development_mode() ) {
+			if ( WC_Connect_Jetpack::is_offline_mode() ) {
 				if ( WC_Connect_Jetpack::is_connected() ) {
 					$message = __( 'Note: Jetpack is connected, but development mode is also enabled on this site. Please disable development mode.', 'woocommerce-services' );
```

Comment:
Jetpack is still mentioned in this user facing message

Reply:
Thanks for catching that! Updated in https://github.com/Automattic/woocommerce-services/pull/2677/commits/e8d635dd82732e22b049b656d99430bd9419ca46.